pkgs/development/python-modules/uproot/default.nix

```diff
@@ -15,32 +16,42 @@
 
 buildPythonPackage rec {
   pname = "uproot";
-  version = "4.3.6";
-  format = "setuptools";
+  version = "5.0.0";
+  format = "pyproject";
 
   disabled = pythonOlder "3.7";
 
   src = fetchFromGitHub {
     owner = "scikit-hep";
-    repo = "uproot4";
+    repo = "uproot5";
     rev = "refs/tags/v${version}";
-    hash = "sha256-Te4D2tHVD5fD8DH2njjQMGnTUvLQdcGBzApklnGn6g8=";
+    hash = "sha256-3i6AUQKKWu5KOtfLdLK2LpgkHyiqG30Eiyy6OH4fjUY=";
   };
 
+  # ERROR: Unknown config option: timeout
+  postPatch = ''
+    substituteInPlace pyproject.toml \
+      --replace "timeout = 600" ""
+  '';
```

Comment:
Do the tests work after adding `pytest-timeout` to `checkInputs` instead of replacing this option?